src/pages/Register/Register.js

```diff
@@ -0,0 +1,135 @@
+import { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowLeft } from '@fortawesome/free-solid-svg-icons';
+import './Register.scss';
+
+const validEmail = /^([a-z0-9_\\.-]+)@([\da-z\\.-]+)\.([a-z\\.]{2,6})$/;
+const validPassWord = /^(?=.*?[0-9])(?=.*?[#?!@$ %^&*-]).{8,}$/;
+
+const Register = () => {
+  const [checkEmail, setCheckEmail] = useState('');
+  const [checkPw, setCheckPw] = useState('');
+  const [doubleCheckPw, setDoubleCheckPw] = useState('');
```

Comment:
check는 동사니까 input 값을 받는 state의 목적으로는 조금 어색해보이네요
```suggestion
  const [email, setEmail] = useState('');
  const [password, setPassword] = useState('');
  const [rePassword, setRePassword] = useState('');
```

Reply:
명심하겠습니다, 감사합니다 👍🏻